adam-core/src/test/scala/org/bdgenomics/adam/rdd/SortedGenomicRDDSuite.scala

```diff
@@ -313,6 +314,7 @@ class SortedGenomicRDDSuite extends SparkFunSuite {
     assert(h.count == i.count)
   }
 
+  /*
```

Comment:
We can just test that each partition contains the same data on `z` and `t`. If you'd prefer, I can create a pull request against this branch.